src/actions/constants.js

```diff
@@ -0,0 +1 @@
+// This is a file to list action types
```

Comment:
I don't know if you should commit empty files to lay structure out... I think generally, you should only be introducing code that **does something**. I get the good intentions here, but it could lead to bloat, and down the line, perhaps the team will want to organize differently.

Reply:
I appreciate the feedback Eliot, I'll re-submit this without those files.

Reply:
Files have been removed along with any extra code